polaris-service/src/main/resources/META-INF/hk2-locator/default

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+[org.apache.polaris.service.auth.DefaultPolarisAuthenticator]S
+contract={org.apache.polaris.service.auth.BasePolarisAuthenticator}
+name=default
+qualifier={jakarta.inject.Named}
+
+[org.apache.polaris.service.auth.TestInlineBearerTokenPolarisAuthenticator]S
+contract={io.dropwizard.auth.Authenticator}
+name=test
+qualifier={jakarta.inject.Named}
```

Comment:
I'm not sure I understand the purpose of this declaration. It seems rather redundant to me. The class of `TestInlineBearerTokenPolarisAuthenticator` already has the `@Named("test")` annotation and implements `Authenticator`. This file does not appear to add any new information to what is already known from class files :thinking:

Reply:
This is something that would normally be generated at build time by scanning the build artifacts. But that requires using the HK2-specific annotations (`@Contract` and `@Service`), which I think we agreed to avoid